src/PlaywrightSharp.Chromium/Protocol/DeserializerHelper.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace PlaywrightSharp.Chromium.Protocol
+{
+    internal static class DeserializerHelper

```

Comment:
@kblok  this is the magic that causes the deserialization to work. something similar will be done for the event responses